handler/handler.go

```diff
@@ -39,7 +42,10 @@ func handle(request types.Request) {
 			Name: proto.String("Eremetic"),
 			User: proto.String(""),
 		},
-		Scheduler: scheduler,
+		Scheduler:        scheduler,
+		BindingAddress:   net.ParseIP("0.0.0.0"),
```

Comment:
I might make sense to use that. I think this should be the same as the default but I have no idea why it suddenly needs to specified :confounded: